Arithmetic.pp

```diff
@@ -65,7 +66,10 @@
     ternary() ( ::times:: #multiplication expression() )?
 
 ternary:
-    term() ( ::div:: #division expression() )?
+    quaternary() ( ::div:: #division expression() )?
+
+quaternary:
+    term() ( ::pow:: #power expression() )?
```

Comment:
So, do we have the same associativity than in PHP? If yes, why not using `**`? We will be able to fix test suites directly too.